sycl/doc/PreprocessorMacros.md

```diff
@@ -19,6 +19,13 @@ This behavior is not following the SYCL spec since `constant_ptr` conversions to
 the underlying pointer types return pointers without any additional qualifiers
 so it's disabled by default.
 
+
+### `DISABLE_SYCL_INSTRUMENTATION_METADATA`
+
+This macro is used to disable passing of code location metadata to `submit*`
+methods of `cl::sycl::queue` class.
```

Comment:
I think the intent of this macro is much wider than `cl::sycl::queue` class. Probably we should generalize the wording.

Reply:
Done.